config/Cargo.toml

```diff
@@ -20,6 +20,7 @@ eyre = "0.6.5"
 ethers-core = { git = "https://github.com/gakonst/ethers-rs", default-features = false }
 ethers-solc = { git = "https://github.com/gakonst/ethers-rs", default-features = false, features = ["async", "svm-solc"] }
 Inflector = "0.11.4"
+number_prefix = "0.4.0"
```

Comment:
do we really need this? I think it'd be safe to assume the file name is a block num and try `.parse()`

Reply:
I use this dependency for human readable units, kB and MB, as i am not aware of a better way to implement it

Reply:
ah gotcha, let's keep this!